src/java/org/apache/cassandra/tools/nodetool/Assassinate.java

```diff
@@ -17,27 +17,25 @@
  */
 package org.apache.cassandra.tools.nodetool;
 
-import static org.apache.commons.lang3.StringUtils.EMPTY;
-import io.airlift.airline.Arguments;
-import io.airlift.airline.Command;
-
 import java.net.UnknownHostException;
 
 import org.apache.cassandra.tools.NodeProbe;
-import org.apache.cassandra.tools.NodeTool.NodeToolCmd;
+import picocli.CommandLine;
```

Comment:
This was done automatically by the IDE because we have several similar libraries in the classpath and these names are clashes. I've fixed it manually.